components/scream/src/share/util/scream_family_tracking.hpp

```diff
@@ -0,0 +1,170 @@
+#ifndef SCREAM_FAMILY_TRACKING_CLASS
+#define SCREAM_FAMILY_TRACKING_CLASS
+
+#include "ekat/std_meta//ekat_std_enable_shared_from_this.hpp"
```

Comment:
You've got a double slash here.

Reply:
Argh, I saw that and wanted to get rid of it, then I forgot. You've got a 25/20 vision, sir.